packages/gatsby-theme-vtex/src/sdk/pixel/GTM/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, Fragment } from 'react'
+
+import { isServer } from '../../../utils/env'
+import { useLazyScript } from '../../lazyScript/useLazyScript'
+import { usePixelEvent } from '../usePixelEvent'
+import { handler } from './handler'
+
+if (!isServer) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push({ 'gtm.blacklist': ['html'] })
+  window.dataLayer.push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' })
+}
+
+interface Props {
+  gtmId: string
+}
+
+const Provider: FC<Props> = ({ children, gtmId }) => {
+  useLazyScript({
+    src: `https://www.googletagmanager.com/gtm.js?id=${gtmId}`,
+    id: gtmId,
+    timeout: 5.5e3, // add script 5.5s after idle to decrease tti
```

Comment:
Any reason to use a double instead of a simple `5500`?

Reply:
Not really, it's just more readable for me, but I think the int may be more performatic, nice !